packages/widgets/src/tabbar.ts

```diff
@@ -777,15 +777,28 @@ export class TabBar<T> extends Widget {
       return;
     }
 
+    // Get focus element that is in focus by the tab key
+    const focusedElement = document.activeElement;
+    // Check if Delete key has been pressed and Delete that tab
+    if (event.key === 'Delete') {
```

Comment:
Why delete? I tested multiple applications (Chrome, Edge, Firefox, VSCode, Gnome Text Editor) and none of them uses Delete to close a focused tab. In my opinion if there was to be a shortcut added:
- it should be added in JupyterLab not in lumino
- it should be explained why the specific default is proposed

Reply:
Hi @krassowski 
From an accessibility point of view documentation states that this functionality is an optional feature for tab elements: [here](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)  
Happy to remove this and create a new PR in Jupyter, thoughts?

Reply:
Since this is a part of a standard I think it is ok to use. Can you the link and rationale to the PR description?

Reply:
Yes, have added in